sdk/identity/identity/CHANGELOG.md

```diff
@@ -17,6 +17,11 @@
   - Added `regionalAuthority` property to `ClientSecretCredentialOptions` and `ClientCertificateCredentialOptions`.
   - If instead of a region, `AutoDiscoverRegion` is specified as the value for `regionalAuthority`, MSAL will be used to attempt to discover the region.
   - A region can also be specified through the `AZURE_REGIONAL_AUTHORITY_NAME` environment variable.
+- The `AzureCliCredential` now receives an `AzureCliCredentialOptions` object, which has the same structure of the `TokenCredentialOptions`, but with an extra optional `tenantId`.
+- The `AzurePowerShellCredential` now receives an `AzurePowerShellCredentialOptions` object, which has the same structure of the `TokenCredentialOptions`, but with an extra optional `tenantId`.
+- All of the credentials now accept `allowMultiTenantAuthentication`, and as long as they allow configuring a `tenantId`, they now also work with the `GetTokenOptions` new property: `tenantId`.
```

Comment:
I kinda want to merge all three of these bullets together, since it is a bit hard to piece together what is supported right now

Reply:
Hmm I’m not sure if I can see clearly how to merge these thee bullets.

Reply:
I don't know, something like "`AzureCliCredential` and `AzurePowerShellCredential` now take an optional `tenantId` property, which enables the use of a new `allowMultiTenantAuthentication` option."

Reply:
well, the issue is that all the credentials do now! With one exception: ManagedIdentityCredential doesn’t take a `tenantId` (yet), this is because we haven’t come up with the solution for that credential across languages